conf/config.level3.neon

```diff
@@ -3,6 +3,7 @@ includes:
 
 rules:
 	- PHPStan\Rules\Arrays\ArrayDestructuringRule
+	- PHPStan\Rules\Arrays\ArrayUnpackingRule
```

Comment:
Can you please adapt this similarly to this commit? https://github.com/phpstan/phpstan-src/pull/1077/commits/7627b52b977480e71254a84f08b01e6251076619

Reply:
Updated.